scipy/optimize/_linprog_ip.py

```diff
@@ -28,13 +28,11 @@
 has_umfpack = True
 has_cholmod = True
 try:
-    import sksparse
-    from sksparse.cholmod import cholesky as cholmod
     from sksparse.cholmod import analyze as cholmod_analyze
 except ImportError:
     has_cholmod = False
 try:
-    import scikits.umfpack  # test whether to use factorized
+    pass  # test whether to use factorized
```

Comment:
This appears to change something. I don't want to dig into whether it matters or not. These solvers are deprecated, so can we just `# noqa`?